packages/components/src/components/SignIn.vue

```diff
@@ -11,30 +11,18 @@
       </div>
 
       <ul class="signin-ctas">
-        <li @click="clickSignInLink('sequence-diagrams')">
-          <a href="https://appmap.io/docs/diagrams/sequence-diagrams">Sequence diagrams</a>
+        <li>
+          <a href="https://appmap.io/docs/navie">AppMap Navie AI</a>
         </li>
-        <li @click="clickSignInLink('trace-views')">
-          <a
-            href="https://appmap.io/docs/diagrams/how-to-use-appmaps.html#view-and-interact-with-the-trace-diagram"
-            >Trace views</a
-          >
+        <li>
+          <a href="https://appmap.io/docs/diagrams">Code Visualizations</a>
```

Comment:
Minor nit pick: consider the casing in `Code Visualizations` vs `Runtime analysis` - we should adhere to a single style.